### ISSUE TITLE
Running plugin outside of integration test

### ISSUE THREAD

**Issue by ShahinKordasti** [ipastusi/wiremock-maven-plugin#6]
When following the instructions and running `mvn verify` the wiremock instance if started properly with the params specified in the pom.xml. But when running just: `mvn wiremock:run` it seems to start a wiremock instance on port 8080, which I assume is the default one, so it seems it is ignoring the params in the pom.xml. How do I start a wiremock instance which is configured in my pom.xml:

```
<plugin>
	<groupId>uk.co.deliverymind</groupId>
	<artifactId>wiremock-maven-plugin</artifactId>
	<version>2.3.0</version>
	<executions>
		<execution>
			<goals>
				<goal>run</goal>
			</goals>
			<configuration>
				<dir>target/classes</dir>
				<params>--port=9923 --verbose --local-response-templating</params>
			</configuration>
		</execution>
	</executions>
</plugin>
```

**Comment by automatictester** [comment]
Hi @ShahinKordasti, thanks for your question. You can achieve what you want by moving configuration section to the plugin scope:

```
<plugin>
	<groupId>uk.co.deliverymind</groupId>
	<artifactId>wiremock-maven-plugin</artifactId>
	<version>2.3.0</version>
	<configuration>
		<dir>target/classes</dir>
		<params>--port=9923 --verbose --local-response-templating</params>
	</configuration>
</plugin>
```

If you run it with `mvn wiremock:run`, I don't think you'll then even need executions section.

Does it solve your problem?

**Comment by ShahinKordasti** [comment]
It did but I was expecting the plugin to start wiremock up and keep it running, so you can run your tests in your IDE. But it just starts and then immediately stops which is not very useful. Is this intended behaviour and, if so, it would be useful to give it a flag, like wait, to keep it running.

**Comment by automatictester** [comment]
Hi @ShahinKordasti if you want to run WireMock outside of Maven lifecycle, why wouldn't you use WireMock Standalone JAR?

**Comment by ShahinKordasti** [comment]
I guess the same reason as why you have embedded tomcat instead of an external one. That way people can run the tests in their IDE and not everyone has to install wiremock as it would be handled by Maven.

**Comment by automatictester** [comment]
Hi @ShahinKordasti, requested functionality is available in version 2.5.0 which has just been released and should be available from Maven Central shortly.
